internals/search_fulltext.py

```diff
@@ -22,6 +22,10 @@
 
 from framework.basehandlers import FlaskHandler
 from internals.core_models import FeatureEntry
+from internals.feature_helpers import (
+    get_future_results,
+    get_entries_by_id_async,
```

Comment:
```suggestion
    get_entries_by_id_async
```
And fix formatting after

Reply:
Done